src/main/scala/se/nimsa/sbx/metadata/MetaDataDAO.scala

```diff
@@ -44,8 +44,8 @@ class MetaDataDAO(val dbConf: DatabaseConfig[JdbcProfile])(implicit ec: Executio
 
   class PatientsTable(tag: Tag) extends Table[Patient](tag, PatientsTable.name) {
     def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
-    def patientName = column[String](DicomProperty.PatientName.name)
-    def patientID = column[String](DicomProperty.PatientID.name)
+    def patientName = column[String](DicomProperty.PatientName.name, O.Length(255))
+    def patientID = column[String](DicomProperty.PatientID.name, O.Length(255))
```

Comment:
Let's keep it at 255 since VARCHAR does not waste space when strings are shorter than max length.